net/dcwifi/dcstad/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2019 EWSI
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dcstad
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ewsi/$(PKG_NAME)/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_HASH:=3b146ea22bc5480d8264c5ea269831d25993673aa90a9e82dc2dc601a111da55
+PKG_MAINTAINER:=Carey Sonsino <careys@edgewaterwireless.com>
+
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/dcstad
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=Routing and Redirection
+  TITLE:=Dual-Channel WiFi client daemon
+  URL:=https://www.edgewaterwireless.com
+  DEPENDS:= +libdcwsocket +libdcwproto
+  PKG_BUILD_DEPENDS:= libdcwsocket libdcwproto
```

Comment:
Wrong place for this. It should be in the main headings.

Reply:
Actually...it makes no sense to have both DEPENDS and PKG_BUILD_DEPENDS. The former implies the latter.

PKG_BUILD_DEPENDS is usually used when a package needs either headers or a static version of a library.